Helper/Images.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Stockbase\Integration\Helper;
+
+use Psr\Log\LoggerInterface;
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Filesystem\Io\File;
+use Magento\Catalog\Model\Product as ProductModel;
+use Stockbase\Integration\Model\ProductImage;
+use Stockbase\Integration\Model\ResourceModel\ProductImage as ProductImageResource;
+use Stockbase\Integration\Model\Config\StockbaseConfiguration;
+
+/**
+ *
+ * Stockbase images helper.
+ *
+ * Class Images
+ * @package Stockbase\Integration\Helper
+ */
+class Images
+{
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+    /**
+     * @var DirectoryList
+     */
+    private $directoryList;
+    /**
+     * @var File
+     */
+    private $file;
+    /**
+     * @var ProductImage
+     */
+    private $productImage;
+    /**
+     * @var ProductImageResource
+     */
+    private $productImageResource;
+    /**
+     * @var ProductModel
+     */
+    private $product;
+    /**
+     * @var StockbaseConfiguration
+     */
+    private $config;
+
+    /**
+     * Images constructor.
+     * @param LoggerInterface $logger
+     * @param StockbaseConfiguration $config
+     * @param ProductImage $productImage
+     * @param ProductImageResource $productImageResource
+     * @param ProductModel $product
+     * @param DirectoryList $directoryList
+     * @param File $file
+     */
+    public function __construct(
+        LoggerInterface $logger,
+        StockbaseConfiguration $config,
+        ProductImage $productImage,
+        ProductImageResource $productImageResource,
+        ProductModel $product,
+        DirectoryList $directoryList,
+        File $file
+    ) {
+        $this->logger = $logger;
+        $this->config = $config;
+        $this->productImage = $productImage;
+        $this->productImageResource = $productImageResource;
+        $this->product = $product;
+        $this->directoryList = $directoryList;
+        $this->file = $file;
+    }
+
+    /**
+     * Saves images array from stockbase for given $ean
+     *
+     * @param array $images
+     *
+     * @return bool
+     */
+    public function saveProductImages($images)
+    {
+        $this->logger->info('Save images process:');
+        $newImagesCount = 0;
+        // get product model:
+        $productModel = $this->product;
+        // get ean attribute:
+        $eanField = $this->config->getEanFieldName();
+        // loop images:
+        foreach ($images as $image) {
+            $this->logger->info('Image URL: '.$image->{'Url'}.' - EAN: '.$image->EAN);
```

Comment:
I'd suggest to lover the log level for messages like this (see also other parts of the code).
The `info` level is too high for messages like this, there will be too many and the user could miss some more important messages. I think it would be better to use `debug` level for this, or completely remove them.